### PR TITLE
Dev front end

### PR DESCRIPTION
**Pull request by GVill683**
Hey Team!
On the second step of last weeks assignment. Im trying to use the GET method on getPosts.js and store it. In my research I found that localStorage and parse are needed to inorder to attain the end result. But unfortunately I just cant seem to get it to work.

From what I recall from this semester, we need to 
1. Return string
2. Parse string
3. Store it 

PLEASE LEAVE COMMENTS BELOW

### CODE REVIEW

**Comment by ayyrickay** [comment]
@GVill683 I think you may have accidentally opened this PR to mary's README code, but you should be trying to merge into master. Can you change the branch in this pull request?

File: .gitignore [hunk 1]
```diff
@@ -1 +1,14 @@
 node_modules/
+
```
Comment (ayyrickay) [inline .gitignore]: There's no need to include additional information in your `.gitignore`, it's not the README.md. I think these notes are better off in your comment on the Pull Request

Comment (ayyrickay) [inline .gitignore]: There's no need to include additional information in your `.gitignore`, it's not the README.md. I think these notes are better off in your comment on the Pull Request

File: server/app.js [hunk 2]
```diff
@@ -44,3 +49,72 @@ const server = app.listen(3000, () => {
     const port = server.address().port
     console.log('Server is listening on port', port)
 })
+
+
+// // require and instantiate express
```
Comment (ayyrickay) [inline server/app.js]: Can you remove all of this commented out code? It's not doing anything for your server, so it's very distracting and makes the PR look bigger than it actually is

File: client/js/getPosts.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        return response.json(obj);
+    }).then((obj) => {
+        console.log(obj);
+    })
+}
+
+    document.querySelector('#shortenbutton').onclick = getPost
+
+
+// const showPosts = (posts) => {
```
Comment (ayyrickay) [inline client/js/getPosts.js]: Extra commented code with no functionality

**Comment by marvelousdesign** [comment]
I agree with Ricky most of his reviews with keeping our code clean of duplicated commented out code or unused code (if you're using it as backup you can always copy it to another place for reference elsewhere not in the branch or repo).

**Pull request closed without merging**